LEGO1/lego/legoomni/include/infocenter.h

```diff
@@ -50,6 +50,9 @@ class InfocenterState : public LegoState {
 
 	void SetUnknown0x74(MxU32 p_unk0x74) { m_unk0x74 = p_unk0x74; }
 
+	// FUNCTION: BETA10 0x10031bd0
```

Comment:
I think we should call this function `HasSignedIn` since it indicates whether the player has signed in at the Infocenter

Reply:
it exists as `HasRegistered` already (should remove that one)